main/main.cpp

```diff
@@ -384,6 +384,7 @@ void Main::print_help(const char *p_binary) {
 	OS::get_singleton()->print("  --no-docbase                                 Disallow dumping the base types (used with --doctool).\n");
 	OS::get_singleton()->print("  --build-solutions                            Build the scripting solutions (e.g. for C# projects). Implies --editor and requires a valid project to edit.\n");
 	OS::get_singleton()->print("  --dump-extension-api                         Generate JSON dump of the Godot API for GDExtension bindings named 'extension_api.json' in the current folder.\n");
+	OS::get_singleton()->print("  --                                           Any argument after this is considered as user-argument and ignord by Godot.\n");
```

Comment:
Fix typo and improve grammar:

```suggestion
	OS::get_singleton()->print("  --                                           Any argument after this is considered as an user-provided argument and is ignored by the engine.\n");
```